DXRando/DeusEx/Classes/DXRInWorldTrigger.uc

```diff
@@ -0,0 +1,30 @@
+class DXRInWorldTrigger extends Trigger;
+
+var name pawnTag;
+var bool enter;
+
+static function DXRInWorldTrigger Create(Actor a, name pawnTag, name tag, bool enter)
+{
+    local DXRInWorldTrigger iwt;
+
+    iwt = a.Spawn(class'DXRInWorldTrigger',, tag);
+    iwt.pawnTag = pawnTag;
+    iwt.enter = enter;
+    iwt.SetCollision(false, false, false);
+
+    return iwt;
+}
+
+function Trigger(Actor other, Pawn instigator)
+{
+    local ScriptedPawn pawn;
+
+    Super.Trigger(other, instigator);
+    foreach AllActors(class'ScriptedPawn', pawn, pawnTag) {
```

Comment:
This should probably iterate over class'Vehicles' as well, just to cover everything that can Enter/Leave the world